### DISCUSSION TITLE
2FA app lost my github account

### DISCUSSION THREAD

**Discussion by esin-gleb** [community/community#139626] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello! I am using MS Authenticator app on my Android smartphone. After switching to a new device, I discovered that the GitHub account link was gone.
My recovery codes was on Notion, witch blocked my Russian account. 
I'm still logged into GitHub Desktop and Intellij. 
So is it possible to restore my account?

**Comment by ebndev** [comment]
Hey there @esin-gleb,

Please refer to the documentation on [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)

We put together a [FAQ](https://github.com/orgs/community/discussions/68240) in case you have any additional questions that may be answered there!

**Reply by esin-gleb** [reply]
@ebndev thanks for your attention!
Neither the documentation nor this FAQ seems to contain any information about the situation I have.
Can I validate my GitHub account using my current login sessions in Intellij or GitHub Desktop?
